index.html

```diff
@@ -14,6 +14,89 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+    <div>
```

Comment:
As mentioned before, I wouldn't have your content sitting in your body. Created another div component to be the parent for all the content.